src/env.json

```diff
@@ -1,5 +1,5 @@
 {
   "API_HOST": false,
   "GRAASP_COMPOSE_HOST": false,
-  "SHOW_NOTIFICATIONS": false
+  "SHOW_NOTIFICATIONS": true
```

Comment:
@pyphilia, I think that these should always be `false` right? But set to `true` in the `.env.{environment}` file which is ignored by Git. Please confirm this with @Julien-Torrent.

Reply:
Yes this should be `false` in this file, and `true` in env.

Reply:
What is the point of this file then? The values in it are always set to `false`.

If we don't set env vars, it uses hard coded defaults in 'config/constants.js'.

There is no point in trying to load the values from `env.json` in the first place.

Reply:
For me it is mainly a schema for the env variables. Since we are not pushing the secrets anymore, unless we pass them between us it's difficult to find all of them right away. It might have other utility, but @juancarlosfarah knows more about it. But you cannot put `true` otherwise it will become more difficult to set it to `false` with env variables.

Reply:
The schema idea is good, but then we should not use them in the code like this :

```
import env from '../env.json';
const {
  GRAASP_COMPOSE_HOST: ENV_GRAASP_COMPOSE_HOST,
  API_HOST: ENV_API_HOST,
  SHOW_NOTIFICATIONS: ENV_SHOW_NOTIFICATIONS,
} = env;
```